src/js/components/Collapsible.js

```diff
@@ -45,19 +45,21 @@ class Collapse extends Component {
       CLASS_ROOT,
       this.props.className
     );
-
     return <div {...this.props} className={classes} />;
   }
 };
 
 class Collapsible extends Component {
   render () {
     const Component = this.props.animate ? TransitionGroup : Box;
+    let collapseProps = Object.assign({}, this.props);
```

Comment:
for some reason, Object.assign is messing up with the animation. When I'm testing this the animation gets laggy.

Can we replace this by the following? 

`const collapseProps = Props.omit(this.props, Object.keys(Collapsible.propTypes));`

Props is available at:

`import Props from '../utils/Props';`